test/e2e/infra/vagrant/playbook/roles/common/tasks/containerd.yml

```diff
@@ -0,0 +1,56 @@
+- name: Get ubuntu release
+  shell: lsb_release -cs
+  register: release
+  changed_when: False
+
+- name: Add an apt signing key for Docker
+  apt_key:
+    url: https://download.docker.com/linux/ubuntu/gpg
+    state: present
+
+- name: Add apt repository for stable version
+  apt_repository:
+    repo: deb [arch=amd64] https://download.docker.com/linux/ubuntu {{ release.stdout }} stable
+    state: present
+
+- name: Configure prereq modules
+  template: 
+    src: templates/containerd_modules.conf.j2
+    dest: /etc/modules-load.d/containerd.conf
+
+- name: Install prereqs
+  modprobe:
+    name: overlay
+    state: present
+
+- name: Install prereqs
+  modprobe:
+    name: br_netfilter
+    state: present
+
+- name: Configure systctl
+  template:
+    src: templates/99-kubernetes-cri.conf.j2
+    dest: /etc/sysctl.d/99-kubernetes-cri.conf
```

Comment:
could we use https://docs.ansible.com/ansible/latest/collections/ansible/posix/sysctl_module.html instead?

it will also take care of reloading the parameters which I am not sure is the case with the current code (equivalent of `sudo sysctl --system`).